frontend/packages/core-auth/src/types/response.ts

```diff
@@ -83,15 +92,30 @@ export class TokenResponse implements TokenResponseConfig {
     return !(TokenResponse.isTokenFresh(this) || !this.refreshToken);
   }
 
+  getExpiryMs(): number {
+    const tenPercentMargin = 0.9;
```

Comment:
Not sure I understand what the `tenPercentMargin` is for. I'd expect this method to give `expiresIn` in milliseconds. Returning 90% of the value isn't clear.